src/definitions_cxx.hpp

```diff
@@ -549,7 +549,7 @@ enum class LFOType {
 	RANDOM_WALK,
 };
 
-constexpr int32_t kNumLFOTypes = util::to_underlying(LFOType::RANDOM_WALK);
+constexpr int32_t kNumLFOTypes = util::to_underlying(LFOType::RANDOM_WALK) + 1;
```

Comment:
Just for reference, this was a bug I spotted while doing other things. It doesn't need to wait for this PR, but it is a bug.